addons/common/functions/fnc_statusEffect_sendEffects.sqf

```diff
@@ -29,21 +29,20 @@ if (isNull _object) exitWith {};
         //We only do anything if the effect has been defined at some point in the game for this unit
         TRACE_2("checking if event is nil",_x,_effectNumber);
         if (_effectNumber != -1) then {
-            private _eventName = format [QGVAR(%1), _x];
```

Comment:
Anyone using the old format will now be broken

```
["myThing", true, []] call ace_common_fnc_statusEffect_addType;
["ace_common_myThing", {systemChat str _this}] call CBA_fnc_addEVentHandler;
[player, "myThing", "reason", true] call ace_common_fnc_statusEffect_set;
```
nothing happens

Reply:
Adding status effects isn't public API, only get/set are

If something was intentionally adding `ace_common_` to a name that feels pretty clear it's only meant to be internally used